shencha-master/shencha-master/agent/extract_agent.py

```
@@ -1,87 +1,231 @@
+                {"role": "user", "content": prompt}
+            ],
+        }
+
+        try:
+            response = await send_async_request(url, headers, data)
+            if not response or 'choices' not in response:
+                raise ValueError("Invalid API response format")
+                
+            content = response['choices'][0]['message']['content']
+            if content.startswith("```json"):
+                content = content.strip("```json").strip("```")
```

Ruff (0.12.2)

Using `.strip()` with multi-character strings is misleading

(B005)